cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTableAdmin.java

```diff
@@ -186,14 +186,13 @@ private boolean tableExistsInternal(Connection connection, StructuredTableId tab
   }
 
   // Add new non-primary key columns to table
-  private void addColumns(Connection connection, StructuredTableSchema schema)
+  private void addColumns(Connection connection, StructuredTableSchema newSchema, StructuredTableSchema oldSchema)
```

Comment:
nit: use `existingSchema` to be consistent with other methods.

Reply:
updated, pls review